gapic/src/main/com/google/gapid/models/Settings.java

```diff
@@ -56,12 +56,11 @@
   public Point windowLocation = null;
   public Point windowSize = null;
   public boolean hideScrubber = false;
-  public boolean hideLeft = false;
-  public boolean hideRight = false;
   public int splitterTopHeight = 200;
-  public String[] leftTabs = new String[0], centerTabs = new String[0], rightTabs = new String[0];
+  public int[] tabWeights = new int[] { 1, 3, 1 };
+  public String tabStructure = "g3;f1;f1;f2";
```

Comment:
This is a "magic string" used as a small DSL (see other comment above), I think this deserves at least a small comment to point out the meaning / role of this string.